integration-tests/terraform/eks/deployment/main.tf

```diff
@@ -0,0 +1,139 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We dont need the extra level of `deployment` under `eks` for this folder.

Reply:
Updated in revision